d2core/d2audio/ebiten/ebiten_audio_provider.go

```diff
@@ -36,6 +39,8 @@ type AudioProvider struct {
 	lastBgm      string
 	sfxVolume    float64
 	bgmVolume    float64
+
+	logger *d2util.Logger
 }
```

Comment:
why not just embed the logger without the name? then we can just use `eap.Info(message)` instead of `eap.logger.Info(message)`